src/eval_suite.py

```diff
@@ -18,11 +18,11 @@ def run_eval(config: Struct):
 
     AutoConfig.register("retnet", RetNetConfig)
     AutoConfig.register("custom_transformer", DecoderConfig)
-    AutoConfig.register("longnet", DecoderConfig)
-    AutoModel.register(DecoderConfig, LongNetModelHF)
+    AutoConfig.register("longnet", LongNetConfig)  # <-- this breaks now bc it overrides custom transformer
```

Comment:
Is this comment still relevant? It would be nice to have this line before the RetNet and Transformer configuration lines, following the pattern of the two sets of three configuration lines below this one.

Reply:
This comment is no longer relevant, that was a note to myself as I was working through the issue. Fixed in [7aaf18a](https://github.com/DRAGNLabs/301r_retnet/commit/7aaf18aea0e63d95ff9b8755112ebabc44b5c30d).